### PR TITLE
PEP-8 client_lib/pulp/.

### PR DESCRIPTION
**Pull request by rbarlow**
This cannot be merged until the following pull requests are merged:

https://github.com/pulp/pulp_docker/pull/61
https://github.com/pulp/pulp_openstack/pull/18
https://github.com/pulp/pulp_ostree/pull/20
https://github.com/pulp/pulp_puppet/pull/167
https://github.com/pulp/pulp_python/pull/24
https://github.com/pulp/pulp_rpm/pull/658

### CODE REVIEW

**Comment by beav** [comment]
All mentioned PRs are commented on. I will re-review this PR after everything else is updated/merged.

**Comment by rbarlow** [comment]
Thanks! All but one are merged, and that one is still waiting on an LGTM.

**Comment by beav** [comment]
All sub-PRs should be merged at this pint but I suspect you may need a rebase.

**Comment by rbarlow** [comment]
@beav if you don't mind, I'd prefer to rebase after the LGTM. The PEP-8 PRs constantly cause merge conflicts with each other, since they all edit run-tests.py. If I had to rebase to get review, I'd have to rebase all outstanding PRs every time I merged one, which is a lot of work.

All other PRs references are now merged, so this one is ready for review.

File: client_lib/pulp/client/extensions/exceptions.py [hunk 1]
```diff
@@ -12,16 +12,16 @@
...
-from socket import error as socket_error
+from _socket import gaierror
 from gettext import gettext as _
+from socket import error as socket_error
 
 from M2Crypto import X509
 from M2Crypto.SSL.Checker import WrongHost
 
-from pulp.bindings.exceptions import *
+from pulp.bindings import exceptions as bindings_exceptions
```
Comment (beav) [inline client_lib/pulp/client/extensions/exceptions.py]: :sparkling_heart:

**Comment by beav** [comment]
LGTM

**Pull request merged**